docs/source/release_notes/release-notes.rst

```diff
@@ -42,8 +42,9 @@ develop
     *    - Type
          - Change
 
-    *    -
-         -
+    *    - |improved|
+         - Improved heap usage during heavy DBKVS querying
+           (Pull Request <https://github.com/palantir/atlasdb/pull/1560>`__)
```

Comment:
Would be nice if we could quantify this, though I'd be happy to get this in anyway - this looks like it should address the flag in #735 and we can add the magnitude of the improvement before we next release